### ISSUE TITLE
How to retreive  weather for 4 cities ??

### ISSUE THREAD

**Issue by graememilne** [fcambus/ansiweather#151]
Is it possible to show weather for say 4 cities ??
If so what is the command for it ,Please

**Comment by chicks-net** [comment]
I don't think so.  You could be the command into a file 4 times and you've got a script!

**Comment by graememilne** [comment]
Found this sh script which might help someone.................

#!/bin/bash

locations=(
    "London,UK"
    "Manchester,UK"
    "Birmingham,UK"
    "Glasgow,UK"
    "Edinburgh,UK"
    "Bristol,UK"
    # Add more airports as needed
)

for location in "${locations[@]}"
do
    echo "Weather for $location"
    ansiweather -l "$location"
    echo
done
======================================================
Save the script in a file, for example, weather_uk.sh, and then make it executable:

sudo chmod +x weather_uk.sh
=======================================================
Run the script:
./weather_uk.sh
